backend/api/Models/Parcel/ParcelEvaluationModel.cs

```diff
@@ -14,6 +14,8 @@ public class ParcelEvaluationModel : BaseModel
         public float Value { get; set; }
 
         public string Note { get; set; }
+
+        public string Firm { get; set; }
```

Comment:
What is `Firm`?

Reply:
This is for the appraisal firm required in the close out form

Reply:
Ah, the **Appraisal Firm**.  I'd rename to **AppraisalBy**

Reply:
Does that makes sense given that this field is on the evaluation table, and technically could be used by assessments as well?

Reply:
Good point, but `Firm` really doesn't explain itself.  We essentially want it to be self descriptive `ValueFrom`  `SourceOfValue` `Note` would even be better.